data/json/npcs/random_encounters/refugee_caravans.json

```diff
@@ -0,0 +1,296 @@
+[
+  {
+    "type": "npc",
+    "id": "NC_caravan_refugee_A",
+    "//": "Random NPC, can direct you to the free merchants and nearest FEMA camp.",
+    "name_suffix": "refugee",
+    "class": "NC_SCAVENGER",
+    "attitude": 0,
+    "mission": 7,
+    "chat": "TALK_CARAVAN_REFUGEE_A1",
+    "faction": "wasteland_scavengers"
+  },
+  {
+    "type": "npc",
+    "id": "NC_caravan_refugee_B",
+    "//": "Random NPC, can be recruited away.",
+    "name_suffix": "refugee",
+    "class": "NC_SCAVENGER",
+    "attitude": 0,
+    "mission": 8,
+    "chat": "TALK_CARAVAN_REFUGEE_B1",
+    "faction": "wasteland_scavengers"
+  },
+  {
+    "type": "npc",
+    "id": "NC_caravan_refugee_C",
+    "//": "Random NPC, demands heroin and can accept.",
+    "name_suffix": "refugee",
+    "class": "NC_SCAVENGER",
+    "attitude": 0,
+    "mission": 7,
+    "chat": "TALK_CARAVAN_REFUGEE_C1",
+    "faction": "wasteland_scavengers"
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_CARAVAN_REFUGEE_A1",
+    "dynamic_line": "Please don't hurt us, we're just trying to get to safety.",
+    "speaker_effect": { "effect": { "npc_first_topic": "TALK_CARAVAN_REFUGEE_A2" } },
+    "responses": [
+      {
+        "text": "What if I want to hurt you?  I'm so good at it and I've had so much practice.",
```

Comment:
This is a little edgy.

Reply:
Thinking it over, this might be appropriate if locked to psychopath and the other option was denied if you were a psychopath. I'd also expect that to lock out the "further conversation" line with refugee B.

Reply:
Done, leaving refugee B the same on the theory that all psychopaths are sexy https://www.psychologytoday.com/us/blog/attraction-evolved/201910/are-psychopaths-sexy

Reply:
The logic was that they wouldn't ask how they are doing because they don't (can't?) care. e.g. they can't reach `TALK_CARAVAN_REFUGEE_B1_Job`, but they can still ask to join.